model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -329,6 +331,35 @@ object WorkspaceShardStates {
   case object Unknown extends WorkspaceShardState
 }
 
+object WorkspaceType {
+  /**
+    * Represents the system-of-record for a workspace
+    * rawls workspaces are owned and managed by rawls
+    * mc workspaces are owned and managed by workspace manager
+    */
+  sealed trait WorkspaceType extends RawlsEnumeration[WorkspaceType] {
+    override def toString: String = WorkspaceType.toString(this)
+    override def withName(name: String): WorkspaceType = WorkspaceType.withName(name)
+  }
+
+  def withName(name: String): WorkspaceType = name.toLowerCase match {
+    case "rawls" => RawlsWorkspace
+    case "mc" => McWorkspace
```

Comment:
What does "mc" stand for?

Reply:
"Mulit-cloud"

Reply:
I see. Could you add a comment here that "mc" stands for "multi-cloud"? Somehow I feel better about abbreviations when I know what they stand for. :)

Reply:
I made a suggestion in the documentation block.